akka-http-core/src/main/scala/akka/http/javadsl/model/SslSession.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model
+
+import javax.net.ssl.SSLSession
+
+trait SslSession {
```

Comment:
A bit hairy to have `SslSession` and `SSLSession` in the same context.
Would `SslSessionValue` be too ugly?

Reply:
I'd prefer that as well.

Reply:
Hmm, 'value' is a bit ugly - maybe `SslSessionInfo`?

Reply:
Yep, better and more in line with what we had before.